src/olympia/abuse/models.py

```diff
@@ -1256,6 +1260,12 @@ def notify_reviewer_decision(
         action_helper = self.get_action_helper(
             overridden_action=overridden_action, appealed_action=appealed_action
         )
+
+        # TODO: generalize this hack -we shouldn't need to special case specific actions
+        # This will be rewritten for https://mozilla-hub.atlassian.net/browse/AMOENG-1125
```

Comment:
It's the epic

Reply:
Hmmm, yeah I guess we don't have a solution for that in github then. Not a huge fan of jira links in the code since that is not public info but I guess there's no alternative? Only think I could think of is creating a github issue that links to the epic, but that's the same problem with more steps 😆 🤷